src/workerd/io/compatibility-date.capnp

```diff
@@ -749,4 +749,11 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $compatEnableDate("2025-05-19")
       $compatDisableFlag("disable_navigator_language");
   # Enables Navigator.language API.
+
+  enableRequestSignal @87 :Bool
```

Comment:
@kentonv How do you feel about making the `enable_request_signal` flag not be experimental? So no passthrough, no RPC AbortSignal, just able to set a handler for client disconnect.